### PR TITLE
Feature/setup semantic release

### PR DESCRIPTION
**Pull request by TheUnderScorer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,80 +1,135 @@
...
+  },
+  "author": "FingerprintJS, Inc (https://fingerprint.com)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/fingerprintjs/fingerprintjs-pro-react/issues"
+  },
+  "homepage": "https://github.com/fingerprintjs/fingerprintjs-pro-react#readme",
+  "dependencies": {
+    "@fingerprintjs/fingerprintjs-pro-spa": "^0.4.0",
+    "tslib": "2.4.0"
```
Comment (ilfa) [inline package.json:46]: Why it is not devDependency?

File: contributing.md [hunk 2]
```diff
@@ -49,7 +49,5 @@ yarn test:dts
 ```
 
 ### How to publish
-- Create a new branch
-- Run `yarn release:(major|minor|patch)` depending on the version you need
-- Make a pull request
-- After merging the pull request into the main branch and after successful tests, GitHub Action will publish a new version to the npm
\ No newline at end of file
+
+The library is automatically released and published to NPM on every push to the main branch if there are relevant changes. The workflow must be approved by one of the maintainers, first.
```
Comment (ilfa) [inline contributing.md:53]: Add please more information about release mechanism with the links to the documentation for all of used instruments

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -1,41 +1,29 @@
...
-    name: npm-publish
-    runs-on: ubuntu-latest
+  release:
     environment: production
+    runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Cache
-        uses: actions/cache@v2
```
Comment (ilfa) [inline .github/workflows/release.yml:18]: Why we don't need cache anymore?

File: package.json (hunk 1 above)
Comment (TheUnderScorer) [reply package.json:46]: Done!

File: .github/workflows/release.yml (hunk 3 above)
Comment (TheUnderScorer) [reply .github/workflows/release.yml:18]: Must have missed it, restored

**Pull request merged**